src/proto/mgmt/svc.proto

```diff
@@ -108,6 +119,7 @@ message GetAttachInfoResp {
 	string sys = 6;			// Name of the DAOS system
 	repeated RankUri secondary_rank_uris = 7; // Rank URIs for additional providers
 	repeated ClientNetHint secondary_client_net_hints = 8; // Hints for additional providers
+	map<uint32,FabricInterfaces> numa_fabric_interfaces = 9; // Usable fabric interfaces by NUMA node (populated by agent)
```

Comment:
I'm not entirely sure in what format @mchaarawi intends to save it on the libdaos side. Maybe it would be better to just have an array of FabricInterface messages? In the generated C code, it looks like it's not actually presented as a hash table on that side.

Reply:
There is a global variable struct that is populated and now recently cached on daos_int() when we run the first get_attach_info that upcalls to the agent (https://github.com/daos-stack/daos/blob/master/src/mgmt/cli_mgmt.c#L31). I would think this is where we will save that info (or is it now saved there already?) and query it from the C management layer anytime we want to create a new network context to select the interface we want to use for that numa node where the context create is being called from.

I guess having a small example (just in a draft PR not to be landed) to update src/mgmt/cli_mgmt.c that shows what these fields in that struct would look like in C would be helpful. but im sure i can figure it out once i start implementing the original ticket to look at this map.